projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/class-verbum-comments.php

```diff
@@ -101,7 +101,7 @@ function () {
 	 */
 	public function get_form_action() {
 		return is_jetpack_comments() ?
-			wp_json_encode( esc_url_raw( http() . '://' . JETPACK_SERVER__DOMAIN . '/jetpack-comment/' ) ) : site_url( '/wp-comments-post.php' );
+			wp_json_encode( esc_url_raw( http() . '://' . JETPACK_SERVER__DOMAIN . '/jetpack-comment/' ), JSON_UNESCAPED_SLASHES ) : site_url( '/wp-comments-post.php' );
```

Comment:
This seems all sorts of weird. I can't find any code path anywhere where the `wp_json_encode()` would be correct. As best I can tell, it probably only works because WordPress's `esc_url()` blindly strips out the backslashes and quotes that `wp_json_encode` is adding, and the string won't ever contain anything that `json_encode` will hex-encode.
```
> echo json_encode( "https://example.org/foo.php" );
"https:\/\/example.org\/foo.php"
> echo esc_url( json_encode( "https://example.org/foo.php" ) );
https://example.org/foo.php
```

Unfortunately neither https://github.com/Automattic/jetpack/pull/35443/commits/d58eb59ccf35c56e12456c3ec794cfa84b352256 nor any comments on #35443 give any indication of why the `wp_json_encode()` was added here in the first place.

Reply:
Flagging in case @heavyweight has any comments on this, but either way I won't consider it a blocker.

Reply:
Do you have any context as to why it was written this way?

Reply:
@heavyweight I've updated the PR to remove the json_encode here. Can you check the cases you mentioned with this change?